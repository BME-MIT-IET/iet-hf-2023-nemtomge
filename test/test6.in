loadmap map.json 
start V1 
set Gencode LA1 Immunity 5 5 
set Scientist V1 LA1 
set Numgencodes 4
give Gencode V1 Stun 10 10 
give Gencode V1 Dementia 10 10 
give Gencode V1 Craziness 10 5 
touch V1
1
exit
