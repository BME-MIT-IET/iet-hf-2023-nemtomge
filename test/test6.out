Map from file map.json loaded
The game has been started. Players are: V1 
Genetic Code Immunity aminoacid: 5 nucleotide: 5 has been placed on field LA1 
Scientist V1 set on field LA1
Number of Genetic Codes is set to 4
Scientist V1 has been given gencode Stun aminoacid: 10 nucleotide: 10 
Scientist V1 has been given gencode Dementia aminoacid: 10 nucleotide: 10 
Scientist V1 has been given gencode Craziness aminoacid: 10 nucleotide: 5 
Choose one to pick: 
0 - cancel
1 - Immunity(3)
Scientist V1 won the game
